src/lev-report/DateRange.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+  import React from 'react';
```

Comment:
I think this extra space should be removed.

Reply:
Done